File: A1/A1-004/A1-004.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input score
    int exercise, midterm, finals;
    cin >> exercise >> midterm >> finals;
    // Output
    if(exercise >= 5 && midterm >= 20 && finals >= 25)
    {
        cout << "pass";
    }
    else { cout << "fail"; }
    return 0;
}

File: A1/A1-004/A1-004.py
# --------------------------------------------------
# File Name : A1-004.py
# Problem   : Exam Result
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input scores
exercise = int(input())
midterm = int(input())
finals = int(input())

# Output
if exercise >= 5 and midterm >= 20 and finals >= 25:
    print("pass")
else:
    print("fail")
